recipe/meta.yaml

```diff
@@ -36,15 +31,15 @@ requirements:
     - patch     # [not win]
     - m2-patch  # [win]
   host:
-    - cairo >=1.15.10
-    - flake8  # for testing during the build
-    - glib
-    - pytest  # for testing during the build
+    - cairo 1.16.0
+    - flake8 6.0.0 # for testing during the build
+    - glib {{glib}}
+    - pytest 7.1.2 # for testing during the build
     - python
     - setuptools
     - wheel
   run:
-    - cairo
+    - cairo >=1.15.10
```

Comment:
```suggestion
    - cairo >=1.16.0,2.0a0
```

Reply:
https://github.com/pygobject/pycairo/blob/v1.23.0/meson.build#L12
Has `cair_version_req = '>=1.15.10'` can I ask where you see `>=1.16.0,2.0a0` ?

Reply:
Just because our cbc has version 1.16.0 and cairo has a run export on the major version.

Reply:
Ahah. Thanks! Didn't even think to check cbc for cairo